### ISSUE TITLE
Does not return the correct rank for a double pair

### ISSUE THREAD

**Issue by codeKonami** [brunoscopelliti/poker-has#1]
```js
whatIs([
  {rank:'K', type:'H'},
  {rank:'K', type:'D'},
  {rank:'Q', type:'S'},
  {rank:'Q', type:'C'},
  {rank:'5', type:'C'}
]);

// return { strength: 2, rank: 'Q', kickers: [ 'K', '5' ] }
// should return a rank: 'K'
```

**Comment by brunoscopelliti** [comment]
Good spot.
I'm not sure when I can get this fixed... anyway if you would give it a try, there should be something wrong here:

```js
      if (cardsRank.indexOf(firstPairRank) > cardsRank.indexOf(firstPairRank)){
        doublePairResult.rank = firstPairRank;
        doublePairResult.kickers = [secondPairRank].concat(kicker);
      }
      else {
        doublePairResult.rank = secondPairRank;
        doublePairResult.kickers = [firstPairRank].concat(kicker);
      }
```

Also, feel free to add a test case.
Thank you!

**Comment by brunoscopelliti** [comment]
So it was just a typo, passed unnoticed.
I've published `poker-has@2.1.1`.

@codeKonami thank you very much for having reported this issue!
